frontend/src/components/HeaderUserMenu.js

```diff
@@ -142,6 +142,7 @@ function HeaderUserMenu() {
       buttonText={user.name}
       showApplyButton={false}
       direction="left"
+      className="no-print"
```

Comment:
I assume this is the hiding of the initial on print. Not sure if it should be mentioned in the testing steps.

Reply:
Whoops, I didn't notice this PR includes some stuff I did yesterday. I'm straightening out my git state locally right now, but I updated the description to more accurately represent what changes this PR includes.